cloudinit/config/cc_seed_random.py

```diff
@@ -69,6 +69,7 @@
             """
         ),
     ],
+    "activate_by_schema_keys": [],
```

Comment:
Yes because we fall through to do work related to datasource.metadata too on Azure.